src/test/java/org/fundacionjala/coding/Abner/NumberMapTest.java

```diff
@@ -0,0 +1,120 @@
+package org.fundacionjala.coding.Abner;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * This class test the numbers.
+ * Created by Samuel on 24/08/2017.
+ */
+public class NumberMapTest {
+    /**
+     * This bar set for all class.
+     */
+    private NumberMap map;
+
+    /**
+     * this function setup de var for all class.
+     */
+    @Before
+    public void setup() {
+        map = new NumberMap();
+    }
+
+    /**
+     * This test check whit nine value.
+     */
+    @Test
+    public void testMapNumberCheckValue() {
```

Comment:
please declare variables for expectedResult and actualResult as final in your tests